app/code/core/Mage/Adminhtml/Block/Template.php

```diff
@@ -34,6 +34,13 @@
 class Mage_Adminhtml_Block_Template extends Mage_Core_Block_Template
 {
     /**
+     * Block event prefix
+     *
+     * @var string
+     */
+    $_eventPrefix = 'adminhtml';
```

Comment:
Is this the correct event prefix for this class? It seems like `adminhtml_template` might be a better choice, n o?